src/sentry/static/sentry/app/components/group/sidebar.tsx

```diff
@@ -14,20 +14,35 @@ import GroupReleaseStats from 'app/components/group/releaseStats';
 import GroupTagDistributionMeter from 'app/components/group/tagDistributionMeter';
 import GuideAnchor from 'app/components/assistant/guideAnchor';
 import LoadingError from 'app/components/loadingError';
-import SentryTypes from 'app/sentryTypes';
 import SuggestedOwners from 'app/components/group/suggestedOwners/suggestedOwners';
 import withApi from 'app/utils/withApi';
+import {
+  Event,
+  Environment,
+  Group,
+  Organization,
+  Project,
+  TagWithTopValues,
+} from 'app/types';
 
-class GroupSidebar extends React.Component {
-  static propTypes = {
-    api: PropTypes.object,
-    organization: SentryTypes.Organization,
-    project: SentryTypes.Project,
-    group: SentryTypes.Group,
-    event: SentryTypes.Event,
-    environments: PropTypes.arrayOf(SentryTypes.Environment),
-  };
+type Props = {
+  api: Client;
+  organization: Organization;
+  project: Project;
+  group: Group;
+  event: Event;
```

Comment:
you are having a type issue here.. it says " null' is not assignable to type 'Event'"

Reply:
Hmm good catch. It was set to `any` type by a JSX HOC.